<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COMSettings</class>
 <widget class="QDialog" name="COMSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMSettings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="stopBitsLabel">
     <property name="text">
      <string>Stop bits:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="dataBitsLabel">
     <property name="text">
      <string>Data bits:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="parityLabel">
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Serial_Port">
     <property name="text">
      <string>Serial Port:</string>
     </property>
     <property name="buddy">
      <cstring>Port</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Baud_Rate_2">
     <property name="text">
      <string>Baud Rate:</string>
     </property>
     <property name="buddy">
      <cstring>Baud_Rate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="parityBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="flowControlLabel">
     <property name="text">
      <string>Flow control:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="stopBitsBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="dataBitsBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="Baud_Rate">
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Port">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="flowControlBox">
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
